analyzers/tests/SonarAnalyzer.UnitTest/TestCases/CastShouldNotBeDuplicated.CSharp12.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using Person = (string name, string surname);
+
+class MyClass
+{
+    void AliasType()
+    {
+        var persons = new List<object> { ("Mickey", "Mouse") };
```

Comment:
You can simplify the test case just have a parameter of type object in the method.

Reply:
You don't need a list in the test case at all. you can just get an object and cast that one directly.